projects/angular/src/accordion/stepper/stepper-panel.ts

```diff
@@ -53,7 +57,7 @@ export class ClrStepperPanel extends ClrAccordionPanel implements OnInit {
     ifExpandService: IfExpandService,
     cdr: ChangeDetectorRef
   ) {
-    super(commonStrings, stepperService, ifExpandService, cdr);
+    super(parent, commonStrings, stepperService, ifExpandService, cdr);
```

Comment:
```suggestion
    super(null, commonStrings, stepperService, ifExpandService, cdr);
```

We discussed that this feature is not supported in the stepper. The `ClrAccordionPanel` panel dependency should also be removed.

Reply:
Corrwcted in commit 2cb2400